blockly-core/core/ui/contract_editor/example_view.js

```diff
@@ -52,14 +50,19 @@ Blockly.ExampleView.prototype.initializeTestButton = function (buttonText, iconC
 };
 
 Blockly.ExampleView.prototype.testExample_ = function () {
-  this.resultText.innerHTML = this.onExampleRun_(this.block_);
+  this.contractEditor_.resetExampleViews();
```

Comment:
Just need to take care to make sure we won't need to rely on execution order with multiple listeners down the road